[2025-08-03T11:37:01.189+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-03T11:37:01.209+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cleanup_paused_dags.delete_paused_dags_with_api scheduled__2025-08-03T11:36:00+00:00 [queued]>
[2025-08-03T11:37:01.214+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cleanup_paused_dags.delete_paused_dags_with_api scheduled__2025-08-03T11:36:00+00:00 [queued]>
[2025-08-03T11:37:01.215+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-08-03T11:37:01.224+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): delete_paused_dags_with_api> on 2025-08-03 11:36:00+00:00
[2025-08-03T11:37:01.229+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=552) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-03T11:37:01.229+0000] {standard_task_runner.py:63} INFO - Started process 563 to run task
[2025-08-03T11:37:01.229+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'cleanup_paused_dags', 'delete_paused_dags_with_api', 'scheduled__2025-08-03T11:36:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/cleanup_paused_dags.py', '--cfg-path', '/tmp/tmp7yaod2gc']
[2025-08-03T11:37:01.231+0000] {standard_task_runner.py:91} INFO - Job 103: Subtask delete_paused_dags_with_api
[2025-08-03T11:37:01.278+0000] {task_command.py:426} INFO - Running <TaskInstance: cleanup_paused_dags.delete_paused_dags_with_api scheduled__2025-08-03T11:36:00+00:00 [running]> on host 09e70cadbe97
[2025-08-03T11:37:01.369+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='cleanup_paused_dags' AIRFLOW_CTX_TASK_ID='delete_paused_dags_with_api' AIRFLOW_CTX_EXECUTION_DATE='2025-08-03T11:36:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-03T11:36:00+00:00'
[2025-08-03T11:37:01.369+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-03T11:37:01.383+0000] {cleanup_paused_dags.py:86} INFO - 找到 0 個暫停的 DAG
[2025-08-03T11:37:01.383+0000] {cleanup_paused_dags.py:126} INFO - 完成！成功刪除 0 個 Airflow DAG，API 成功刪除 0 個資料庫記錄
[2025-08-03T11:37:01.383+0000] {python.py:237} INFO - Done. Returned value was: 完成！成功刪除 0 個 Airflow DAG，API 成功刪除 0 個資料庫記錄
[2025-08-03T11:37:01.384+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-03T11:37:01.403+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=cleanup_paused_dags, task_id=delete_paused_dags_with_api, run_id=scheduled__2025-08-03T11:36:00+00:00, execution_date=20250803T113600, start_date=20250803T113701, end_date=20250803T113701
[2025-08-03T11:37:01.442+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-03T11:37:01.463+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-03T11:37:01.465+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
